.github/workflows/blossom-ci.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A workflow to trigger ci on hybrid infra (github + self hosted runner)
+name: Blossom-CI
+on:
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+      inputs:
+          platform:
+            description: 'runs-on argument'     
+            required: false
+          args:
+            description: 'argument'     
+            required: false
+jobs:
+  Authorization:
+    name: Authorization
+    runs-on: blossom 
+    outputs:
+      args: ${{ env.args }}
+      
+    # This job only runs for pull request comments
+    if: |
+         contains( 'anstockatnv,rohitrajani2018,', format('{0},', github.actor)) && 
```

Comment:
Just as a matter of interest, could we check a `MAINTAINERS.md` file or something similar here instead of manually adding users?

Reply:
Kubernetes uses OWNERS files

Reply:
+1 for the OWNERS File check

Reply:
I will work with Rohit and Andrew in an Owners file feature, but for now we will use the blossom-ci file as designed by the NDASH team